submissions/ogennaOgbogu/CSS/contactUsStyleSheet.css

```diff
@@ -0,0 +1,366 @@
+* {
+    box-sizing: border-box;
+}
+
+html,body {
+    width: 100%;
+    height: 100%;
+    margin: 0;
+    padding: 0;
+    overflow-x: auto;
+    overflow-y: auto;
+}
+
```

Comment:
Remove extra space. There are other places where you have more than one blank line between your CSS selectors. Please ensure that you make the necessary changes.